### PR TITLE
Add Quarterly Update, Q4 2022

### PR DESCRIPTION
**Pull request by chrean**
## Summary

Add quarterly update, Q4, 2022.

## Checklist

Check out our [PR guidelines](https://github.com/solidusio/.github/blob/master/CONTRIBUTING.md#pull-request-guidelines) for more details.

The following are mandatory for all PRs:

- [x] I have written a thorough PR description.
- [x] I have kept my commits small and atomic.
- [x] [I have used clear, explanatory commit messages](https://github.com/solidusio/.github/blob/main/CONTRIBUTING.md#writing-good-commit-messages).

The following are not always needed:

- 📖 I have updated the README to account for my changes.
- 📑 I have documented new code [with YARD](https://www.rubydoc.info/gems/yard/file/docs/Tags.md).
- 🛣️ I have opened a PR to update the [guides](https://github.com/solidusio/edgeguides).
- ✅ I have added automated tests to cover my changes.
- 📸 I have attached screenshots to demo visual changes.

### CODE REVIEW

File: source/blog/2023-02-02-solidus-quarterly-roundup-q4-2022.html.markdown [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+This is Gabriel writing, Head of R&D at [Nebulab](www.nebulab.com), where we are helping Solidus evolve toward an even brighter future.
+
+Now off with the introductions, and let’s get started!
+
+
+# What’s new
+
+
+## PayPal Commerce Extension 1.0
```
Comment (kennyadsl) [inline source/blog/2023-02-02-solidus-quarterly-roundup-q4-2022.html.markdown:22]: I think the product is called PayPal Commerce Platform. 

```suggestion
## Solidus PayPal Commerce Platform 1.0
```

File: source/blog/2023-02-02-solidus-quarterly-roundup-q4-2022.html.markdown [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+We promised, and we are delivering. The last time you heard from me, I mentioned we wanted to work on payment extensions, make the main ones officially supported, and make them compatible with the new Starter Storefront, which means that the extensions will work out of the box with Solidus.
+
+PayPal Commerce is just the first step of this process and just came out of beta. So feel free to add the gem to your project, test it, and let us know how it goes.
```
Comment (kennyadsl) [inline source/blog/2023-02-02-solidus-quarterly-roundup-q4-2022.html.markdown:26]:
```suggestion
Solidus PayPal Commerce Platform is just the first step of this process and just came out of beta. So feel free to add the gem to your project, test it, and let us know how it goes.
```

File: source/blog/2023-02-02-solidus-quarterly-roundup-q4-2022.html.markdown [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+## Solidus 3.3 has been released
+
+Honoring our quarterly release cadence for minors, here we are with 3.3.
+
+A few minor but important features have been added; here is an incomplete list with the main ones.
+
+
+### Configurable algorithm to prioritize store credits application
+
+Before v3.3, store credits were always sorted by the priority of their type before being applied to an order during the checkout. Now you can change that behavior by configuring the `Spree::Config.store_credit_prioritizer_class` preference with your own class to define your preferred prioritization algorithm.
```
Comment (kennyadsl) [inline source/blog/2023-02-02-solidus-quarterly-roundup-q4-2022.html.markdown:38]:
```suggestion
Before v3.3, store credits were always sorted by the priority of their type before being applied to an order during the checkout. Now you can change that behavior by configuring the `Spree::Config.store_credit_prioritizer_class` preference with your own class to define your preferred prioritization algorithm. Check out the [Pull Request](https://github.com/solidusio/solidus/pull/4677) that introduced this change for more details.
```

**Review by kennyadsl — COMMENTED** [review]
Left some change suggestions. Let me know your thoughts.

**Comment by chrean** [comment]
Thank you @kennyadsl , those suggestions made sense and I have committed them all.
Do you think we are ready to go now?

**Comment by kennyadsl** [comment]
Yes, not required but it would be great to rebase/squash all commits into a single one before merging.

**Review by kennyadsl — APPROVED** [review]

**Pull request merged**